diff --git a/node_modules/iframely/lib/core.js b/node_modules/iframely/lib/core.js
index 5c01cb3..9694105 100644
--- a/node_modules/iframely/lib/core.js
+++ b/node_modules/iframely/lib/core.js
@@ -467,6 +467,9 @@
                 }
             }
 
+            console.log(JSON.stringify({uri, domain, pluginMatchesByDomains}, null, 2))
+
+
             // In domain debug: add all plugins before domain plugins to make them low priority.
             if (options.mixAllWithDomainPlugin) {
                 addAllGeneric();
